### PR TITLE
Add Post-upgrade actions to release-sop

### PR DESCRIPTION
**Pull request by dskloetd**
# Motivation

The `release-sop` script has been very useful in making it easy to do NNS dapp releases.
But it has been incomplete as it didn't include the post-upgrade steps.

# Changes

Add post upgrade steps to the script.

# Tests

Test json files were updated.
The latest release was made with these changes in the script.

# Todos

- [ ] Add entry to changelog (if necessary).
not necessary

### CODE REVIEW

File: scripts/nns-dapp/release-sop [hunk 1]
```diff
@@ -486,3 +498,31 @@ confirm_cmd git checkout "$BRANCH_NAME"
...
+confirm_cmd "$SOURCE_DIR/bump-patch" --commit
+confirm_cmd gh pr create --draft --title "Update changelog after release" --body "# Motivation
+
+A release has been deployed to production.
+
+# Changes
+
+- Changelog - split out the changes included in the release.
+- Increment the patch version of the nns-dapp."
+confirm_manual "Merge release PR" echo "Have the release PR reviewed and merge it."
```
Comment (mstrasinskis) [inline scripts/nns-dapp/release-sop:526]: "...and merged" ?

**Review by mstrasinskis — APPROVED** [review]
Awesome, thank you!

File: scripts/nns-dapp/release-sop (hunk 1 above)
Comment (dskloetd) [reply scripts/nns-dapp/release-sop:526]: The one reviewing is not necessarily the same as the one merging it?
So you have it reviewed by someone else but then you merge it yourself.
Am I missing something?

Comment (mstrasinskis) [reply scripts/nns-dapp/release-sop:526]: Ah, got it 👍

**Pull request merged**